packages/medusa-js/src/resources/admin/discounts.ts

```diff
@@ -203,11 +205,31 @@ class AdminDiscountsResource extends BaseResource {
 
     if (query) {
       const queryString = qs.stringify(query)
-      path = `/admin/discounts/${discountId}/conditions/${conditionId}?${queryString}`
+      path += `?${queryString}`
     }
 
     return this.client.request("GET", path, undefined, {}, customHeaders)
   }
+
+  /**
+   * @description Add a batch of items to a discount condition
+   */
+  addConditionItemBatch(
```

Comment:
todo(if-minor): Rename to be consistent with Seb's comment
```suggestion
  addConditionResourcBatch(
```